bus-mapping/src/evm/opcodes/returndatacopy.rs

```diff
@@ -41,10 +74,115 @@ impl Opcode for Returndatacopy {
                 // there is no more steps.
             }
         }
+
+        let copy_event = gen_copy_event(state, geth_step)?;
+        state.push_copy(copy_event);
         Ok(vec![exec_step])
     }
 }
 
+fn gen_copy_steps(
```

Comment:
What is the motivation to have `gen_copy_steps`?

Reply:
to set correct memory write value into the context rw table according to copy params, so that the readings afterwards could have right values.